generator/config/projection/elemMatch.yaml

```diff
@@ -3,7 +3,7 @@ name: $elemMatch
 link: 'https://www.mongodb.com/docs/manual/reference/operator/projection/elemMatch/'
 type:
     - projection
-encode: object
+encode: single
```

Comment:
Why was `object` used previously? I assume that would have resulted in incorrect nesting with an additional `query` property (based on the public property in the operator class) between `$elemMatch` and the actual value. Is that correct?

Reply:
At first, it looked like an object with properties being each query operator. But since I introduced the Query object, this is the single value.
You guessed it, with encode=object, we had an exported property named "query".